wowup-electron/src/app/pages/options/options.component.html

```diff
@@ -60,6 +60,24 @@
             category="Options" action="UseHardwareAcceleration" [label]="useHardwareAcceleration">
           </mat-slide-toggle>
         </div>
+        <div class="row hover">
+          <div class="grow">
+            <p>{{'PAGES.OPTIONS.APPLICATION.START_WITH_SYSTEM_LABEL' | translate}}</p>
+            <small class="hint">{{'PAGES.OPTIONS.APPLICATION.START_WITH_SYSTEM_DESCRIPTION' | translate}}</small>
+          </div>
+          <mat-slide-toggle [(checked)]="startWithSystem" (change)="onStartWithSystemChange($event)" appUserActionTracker
+            category="Options" action="StartWithSystem" [label]="startWithSystem">
+          </mat-slide-toggle>
+        </div>
+        <div class="row hover">
+          <div class="grow">
+            <p>{{'PAGES.OPTIONS.APPLICATION.START_MINIMIZED_LABEL' | translate}}</p>
+            <small class="hint">{{'PAGES.OPTIONS.APPLICATION.START_MINIMIZED_DESCRIPTION' | translate}}</small>
+          </div>
+          <mat-slide-toggle [(checked)]="startMinimized" (change)="onStartMinimizedChange($event)" appUserActionTracker
+            category="Options" action="StartMinimized" [label]="startMinimized">
+          </mat-slide-toggle>
+        </div>
```

Comment:
New switches in the Options view
Ideally, I would make the second one disabled depending on the state of the first one. However, I just did not find a way to do that besides using this thing: https://angular.io/api/forms/NgModel. Yet I did not see anyting like this in the markup, so I decided to leave it as is - this does not really hurt anything. Maybe switches' text can be modified to be more clear in this regard.

Reply:
@jliddev Can you please also look into this? Is there anything I missed about how to make this work?

Reply:
Nvm :)